src/main/java/org/example/camunda/process/solution/service/FormService.java

```diff
@@ -48,9 +53,10 @@ public Form findByName(String formKey) throws IOException {
   }
 
   public String getEmbeddedFormSchema(String processName, String processDefinitionId, String formId)
-      throws TaskListException {
+      throws TaskListException, NumberFormatException, OperateException {
     if (!loadEmbeddedFormsFromTasklist && processName != null) {
```

Comment:
The purpose of this test is to allow to load embedded forms from local file (for rapidity) or from tasklist graphql services (last line of the method). It could be a better choice to query Operate APIs rather than Tasklist APIs to get the form (to be discussed) but keeping the option to load embedded forms from local resources should be kept. Perhaps we should rename the loadEmbeddedFormsFromTasklist to loadEmbeddedFormsFromLocalResources and inverse logic for better readibility ?